irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -196,6 +196,29 @@ DROP TABLE IF EXISTS index_by_id_height_asset;
       block_store_->dropAll();
     }
 
+    expected::Result<bool, std::string> StorageImpl::createDatabaseIfNotExist(
+        const std::string &dbname,
+        const std::string &options_str_without_dbname) {
+      auto temp_connection =
+          std::make_unique<pqxx::lazyconnection>(options_str_without_dbname);
```

Comment:
Why via pointer? What's the problem with
`pqxx::lazyconnection temp_connection(options_str_without_dbname)`?